tests/contrib/test_cache.py

```diff
@@ -88,7 +91,7 @@ def test_generic_set_many(self, c):
 
     def test_generic_expire(self, c, fast_sleep):
         assert c.set('foo', 'bar', 1)
-        fast_sleep(2)
```

Comment:
[The uWSGI documentation](http://uwsgi-docs.readthedocs.org/en/latest/Caching.html#the-cache-sweeper-thread) (although otherwise very extensive) is a bit brief about the cache sweeper. As far as I understand it, there is no way to force it to garbage collect.